src/transform/HeaderTransform.css

```diff
@@ -0,0 +1,63 @@
+.pagelib_edit_section_link {
+  float: right;
+}
+
+/* todo: standardize on RTL selectors throughout the codebase. In many places we use .content-rtl. */
+html[dir='rtl'] .pagelib_edit_section_link,
+.content-rtl .pagelib_edit_section_link {
+  float: left;
+}
+
+.pagelib_pronunciation svg,
```

Comment:
Missing transform name in these selectors:

pagelib_pronunciation > pagelib_header_pronunciation
pagelib_divider > pagelib_header_divider

Reply:
Done